README.md

```diff
@@ -175,8 +175,8 @@ var storage = multer.diskStorage({
     cb(null, '/tmp/my-uploads')
   },
   filename: function (req, file, cb) {
-    const uniqueHash = Date.now() + '-' + Math.random().split('.')[1]
-    cb(null, file.fieldname + '-' + uniqueHash)
+    const uniqueSuffix = Date.now() + '_' + (Math.random().toString().split('.')[1])
```

Comment:
Why change from `-` to `_`?

Reply:
I've run into issues where Javascript sees a dash and tries to do math. I figured it was safer, but i can change it back if you want.